### PR TITLE
[web-animations] Update timing interfaces

### PR DESCRIPTION
**Pull request by birtles**
These are the tests for https://github.com/w3c/csswg-drafts/pull/2432

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26010)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/353693275)

File: web-animations/interfaces/Animatable/getAnimations.html [hunk 1]
```diff
@@ -91,13 +93,15 @@
   const div = createDiv(t);
   const animation = div.animate(null, 100 * MS_PER_SEC);
 
-  animation.effect.timing.endDelay = -200 * MS_PER_SEC;
+  animation.effect.updateTiming({
+    endDelay: animation.effect.getTiming().endDelay - 200 * MS_PER_SEC,
```
Comment (stephenmcgruer) [inline web-animations/interfaces/Animatable/getAnimations.html]: I don't follow this change. The previous version just set it to `-200 * MS_PER_SEC`, but this version decrements the existing value by `200 * MS_PER_SEC` - why ?

**Review by stephenmcgruer — APPROVED** [review]
Generally LGTM, a bit difficult to review such a large change. One comment that I don't quite follow.

I notice from this that the new API also makes doing 'double the duration' awkward which is a pity :(.

File: web-animations/interfaces/Animatable/getAnimations.html (hunk 1 above)
Comment (birtles) [reply web-animations/interfaces/Animatable/getAnimations.html]: That's a bug in the patch. Thanks!

**Comment by birtles** [comment]
> I notice from this that the new API also makes doing 'double the duration' awkward which is a pity :(.

Yeah, that operation is definitely harder. I wonder in practice how often you do that, however, and how big a deal it is at the time anyway (React programmers anyway are used to this sort of pattern with `setState()` etc.)

**Pull request merged**